engines/bastion/app/assets/javascripts/bastion/components/formatters/array-to-string.filter.js

```diff
@@ -10,7 +10,13 @@
  */
 angular.module('Bastion.components.formatters').filter('arrayToString', [function () {
     return function (toFormat, separator) {
+        if (!toFormat) {
```

Comment:
Is this change in use in this PR?

Reply:
This may be left over from `bst-checkbox-list` which was removed, let me check

Reply:
Looks like that's the case.  Reverting.